Dockerfile

```diff
@@ -12,12 +12,14 @@ RUN apt update && \
       checkinstall \
       curl \
       libblkid1
+
+WORKDIR /src
 RUN curl -LO https://github.com/amadvance/snapraid/releases/download/v${SNAPRAID_VERSION}/snapraid-${SNAPRAID_VERSION}.tar.gz && \
-      tar -xvf snapraid-${SNAPRAID_VERSION}.tar.gz && \
-      cd snapraid-${SNAPRAID_VERSION} && \
-      ./configure && \
-      make -j4 && \
-      make -j4 check && \
-      checkinstall -Dy --install=no --nodoc && \
-      mkdir /build && \
-      cp *.deb /build/snapraid-from-source.deb
+      tar -xvf snapraid-${SNAPRAID_VERSION}.tar.gz
+WORKDIR /src/snapraid-${SNAPRAID_VERSION}
+RUN ./configure
```

Comment:
These changes seem unnecessary. Running them as part of a single stage is more efficient

Reply:
In fact, looking at the fix, i'm not sure any changes in this file are necessary?